### ISSUE TITLE
Hi,may I directly test my data with "stage2_w_external. ckpt"

### ISSUE THREAD

**Issue by Angelalilyer** [Audio-WestlakeU/ATST-SED#13]
Hi, I don't have any new training data, I am detecting sound events for a small test set. However, in "stage2. yaml", there are a lot of files that need to be filled in, such as "config ["data"] ["test_tsv"]. I only have one test folder containing some audios. How should I proceed with the testing script?

**Comment by Angelalilyer** [comment]
my running script is  python  train_stage2.py  --eval_from_checkpoint  ATST-SED/src/stage_2_wo_external.ckpt
the setting in the stage2.yaml is:
ultra:
  atst_dropout: 0.0
  model_init: ATST-SED/src/stage_1.ckpt
  atst_init: ATST-SED/src/atst_as2M.ckpt
All of these were downloaded from you, but an error occurred after running them：
————————————————————————————————————————————————
desed_training = SEDICT(

stage2_trainer.py 50 __init__
super(SEDICT, self).__init__(*args, **kwargs)

ultra_sed_trainer.py 126 __init__
self.get_weak_student_f1_seg_macro = torchmetrics.classification.f_beta.MultilabelF1Score(

f_beta.py 1033 __init__
super().__init__(

f_beta.py 522 __init__
super().__init__(

stat_scores.py 471 __init__
super(_AbstractStatScores, self).__init__(**kwargs)

metric.py 149 __init__
raise ValueError(f"Unexpected keyword arguments: {', '.join(kwargs_)}")

ValueError:
Unexpected keyword arguments: `compute_on_step`

If you could tell me the correct testing steps, I would greatly appreciate it！！

**Comment by SaoYear** [comment]
Hi, thanks for the interest on our work!

If I understand you correctly, you want to use the model to inference on a small set of wavforms, whose labels are unknown.

In this case, you can refer to [this issue](https://github.com/Audio-WestlakeU/ATST-SED/issues/8).

**Comment by SaoYear** [comment]
Oh, I saw your comments. I will response you in that issue.

**Comment by SaoYear** [comment]
I close this issue because the questions are discussed in another [issue](https://github.com/Audio-WestlakeU/ATST-SED/issues/8).
